Rakefile

```diff
@@ -3,16 +3,85 @@
 require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
 
+RSpec::Core::RakeTask.new(:spec)
+
 $REPO_ROOT = File.dirname(__FILE__)
 $VERSION = ENV['VERSION'] || File.read(File.join($REPO_ROOT, 'VERSION')).strip
 
 Dir.glob('**/*.rake').each do |task_file|
   load task_file
 end
 
-RSpec::Core::RakeTask.new(:test)
-
 RuboCop::RakeTask.new
 
-task default: :test
-task 'release:test' => :test
+task default: :spec
+task 'release:test' => :spec
+
+PRIVATE_KEY = 'spec/fixtures/pem/privatekey.pem'
+CERTIFICATE = 'spec/fixtures/pem/certificate.pem'
+AWS_FIXTURES = FileList['spec/fixtures/json/*.json'].exclude('**/*/invalid_signature.json')
+SIGNABLE_KEYS = %w[
+  Message
+  MessageId
+  Subject
+  SubscribeURL
+  Timestamp
+  Token
+  TopicArn
+  Type
+].freeze
+
+file PRIVATE_KEY do |t|
```

Comment:
If we can't move this logic into the test runners themselves, can we move all of these test fixture/cert tasks to a separate file?

Reply:
I think the test runner comment has been handled.

Why would we move into it a separate file if it's a rake task?